blogger.go

```diff
@@ -0,0 +1,125 @@
+package main
```

Comment:
We're ready to start moving this code into a `blogger` package, aren't we? Then `main.go` can move into a `cmd` subfolder, to build the binary which runs the web server.

Reply:
Yeah, I agree that would make a neat structure.